.teamcity/builds/Common.kt

```diff
@@ -32,6 +32,28 @@ enum class JavaVersion(val version: String, val dockerImage: String) {
   V_17(version = "17", dockerImage = "eclipse-temurin:17-jdk"),
 }
 
+enum class Neo4jVersion(
```

Comment:
I don't see any CE versions in there, is that intentional?

Reply:
Gave it a shot but CE will definitely require a lot of work because of the way our tests work currently, like creating a new database for each test, etc.